net-mgmt/collectd/src/opnsense/mvc/app/controllers/OPNsense/Collectd/forms/general.xml

```diff
@@ -15,7 +15,7 @@
         <id>general.fqdnlookup</id>
         <label>FQDN lookup</label>
         <type>checkbox</type>
-        <help>This will enable the FQDNLookup option.</help>
+        <help>If Hostname is determined automatically this setting controls whether or not the daemon should try to figure out the "fully qualified domain name", FQDN. This is done using a lookup of the name returned by gethostname.</help>
```

Comment:
probably write FQDN into round brackets as it is an abbreviation and remove the quotes.

"This is done by looking up the name by using gethostname."